docs/source/tutorial/sourcing_data/query_sql.rst

```diff
@@ -0,0 +1,53 @@
+======================================================
+Source data stored in a PostgreSQL Database
+======================================================
+
+Scenario
+========
+
+You would like to extract data directly from a PostgreSQL Database.
+
```

Comment:
@nicholasvk how does this look?

```suggestion
.. note::
    Before deciding that you'd like to query a SQL database correctly, instead,
    you may want to consider separately querying your database, saving the
    result of that query to a file, and then using that file as source for an
    extract config. 

    By querying a database directly

        * It's no longer easy to audit the data used as the source for an
          ingest.
        * It's much harder, if not impossible, for non-technical users to know
          what data is used for an ingest
        * Using a database is inherently volatile e.g. rows may be added/
          deleted/ updated.

    Because of the above points, it's important to consider if your needs
    wouldn't be better served by using a static result of your query as
    source data for an ingest instead of directly querying the SQL database. 


```

Reply:
Looks great to me @chris-s-friedman . Thanks for considering my feedback and summarizing it nicely here!